solr/heron/conf/solrconfig.xml

```diff
@@ -1221,220 +1221,120 @@ unless you are sure you always want it.
-             once rev. 1.2.11 is deployed, and also for the other fields
-             below It indexes all the words as suggestions, in addition to
-             the whole field value.  The only reason we have it is because there
-             is an NPE if you don't have any word-based fields.
-        -->
-        <lst name="field">
-          <str name="name">publisher</str>
-          <float name="weight">1.0</float>
-        </lst>
-      </lst>
-      <str name="classname">com.ifactory.press.db.solr.spelling.suggest.MultiSuggester</str>
+    <lst name="suggester">
+      <str name="name">suggester-publisher</str>
+
       <str name="lookupImpl">com.ifactory.press.db.solr.spelling.suggest.SafeInfixLookupFactory</str>
-      <str name="indexPath">${solr.core.instanceDir}/${solr.core.dataDir}/suggest-publisher</str>
+      <str name="indexPath">${solr.core.instanceDir}/${solr.core.dataDir}/suggester-publisher</str>
+      <str name="dictionaryImpl">DocumentDictionaryFactory</str>
+      <str name="field">title</str>
       <str name="suggestAnalyzerFieldType">text_suggest</str>
-      <float name="threshold">0.0</float>
+      <str name="buildOnStartup">false</str>
       <str name="buildOnCommit">false</str>
+      <str name="highlight">true</str>
     </lst>
 
-    <lst name="spellchecker">
-      <str name="name">suggest-infix-isbn</str>
-      <lst name="fields">
-        <lst name="field">
-          <str name="name">isbn</str>
-          <str name="analyzerFieldType">string</str>
-        </lst>
-        <lst name="field">
-          <str name="name">isbn</str>
-          <str name="analyzerFieldType">string</str>
-          <float name="weight">2.0</float>
-        </lst>
-        <lst name="field">
-          <str name="name">publisher</str>
-          <float name="weight">1.0</float>
-        </lst>
-      </lst>
-      <str name="classname">com.ifactory.press.db.solr.spelling.suggest.MultiSuggester</str>
+    <lst name="suggester">
+      <str name="name">suggester-isbn</str>
+
       <str name="lookupImpl">com.ifactory.press.db.solr.spelling.suggest.SafeInfixLookupFactory</str>
-      <str name="indexPath">${solr.core.instanceDir}/${solr.core.dataDir}/suggest-isbn</str>
+      <str name="indexPath">${solr.core.instanceDir}/${solr.core.dataDir}/suggester-isbn</str>
+      <str name="dictionaryImpl">DocumentDictionaryFactory</str>
+      <str name="field">title</str>
       <str name="suggestAnalyzerFieldType">text_suggest</str>
-      <float name="threshold">0.0</float>
+      <str name="buildOnStartup">false</str>
       <str name="buildOnCommit">false</str>
+      <str name="highlight">true</str>
     </lst>
-    
   </searchComponent>
 
-  <!-- A request handler for the Suggester -->
-  <requestHandler name="/suggest/all" class="solr.SearchHandler" startup="lazy">
+  <!-- Request handlers for the SuggestComponent Suggesters -->
+  <requestHandler name="/suggester/all" class="solr.SearchHandler" startup="lazy" >
```

Comment:
This will require a change to the endpoint that Falcon uses for suggestions. Is that the intent?

Reply:
Ah, never mind. We do use `suggester` in Falcon. 👍

Reply:
Yeah, I was a little late on updating the configs used here for testing 😬 (`suggest` was used for the V1 suggestions)